gis/agents_and_networks/src/model/model.py

```diff
@@ -22,19 +22,21 @@ def get_time(model) -> pd.Timedelta:
 
 def get_num_commuters_by_status(model, status: str) -> int:
     commuters = [
-        commuter for commuter in model.schedule.agents if commuter.status == status
+        commuter
+        for commuter in model.agents_by_type[Commuter]
```

Comment:
Why is this `model.agents_by_type` instead of just `model.agents`?

Reply:
Because there are also `Building` agents in the model.